packages/builder/src/builder.js

```diff
@@ -327,12 +327,11 @@ export default class Builder {
     consola.debug('Generating routes...')
 
     if (this._defaultPage) {
-      templateVars.router.routes = createRoutes(
-        ['index.vue'],
-        this.template.dir + '/pages',
-        '',
-        this.options.router.routeNameSplitter
-      )
+      templateVars.router.routes = createRoutes({
+        files: ['index.vue'],
```

Comment:
As of usage change can you please double check that `createRoutes` is not exposed anywhere? (ensuring no one will break by this)

Reply:
BTW nice change :+1:

Reply:
@pi0 I checked, updated tests & snapshots